sdk/ai/azure-ai-resources/azure/ai/resources/entities/__init__.py

```diff
@@ -13,11 +13,13 @@
     AzureAIServiceConnection,
     GitHubConnection,
     CustomConnection,
+    AzureBlobStoreConnection,
 )
 from .mlindex import Index
 from .project import Project
 from .data import Data
 from .configs import AzureOpenAIModelConfiguration
+from azure.ai.ml.entities import ApiKeyConfiguration
```

Comment:
Considering how primitive this object is, I opted to just directly forward it from the v2 SDK without any wrapping.